src/devtools/client/webconsole/components/App.js

```diff
@@ -5,7 +5,7 @@
 
 const React = require("react");
 const PropTypes = require("prop-types");
-const { connect } = require("devtools/client/shared/redux/visibility-handler-connect");
```

Comment:
This was a no-op since all of the panels are in the same iframe and therefor the same visible document